plugins/rtmp-services/data/services.json

```diff
@@ -2846,6 +2846,139 @@
+                },
+                {
+                    "name": "Live Stream Server #13",
+                    "url": "rtmps://live-stream-server-13.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #14",
+                    "url": "rtmps://live-stream-server-14.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #15",
+                    "url": "rtmps://live-stream-server-15.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #16",
+                    "url": "rtmps://live-stream-server-16.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #17",
+                    "url": "rtmps://live-stream-server-17.wavezoo.com:8443/LiveApp"},
+                {
+                    "name": "Live Stream Server #18",
+                    "url": "rtmps://live-stream-server-18.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #19",
+                    "url": "rtmps://live-stream-server-19.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #20",
+                    "url": "rtmps://live-stream-server-20.wavezoo.com:8443/LiveApp"
+                }
+            ],
+            "protocol": "RTMPS",
+            "supported video codecs": [
+                "h264",
+                "hevc"
+            ],
+            "supported audio codecs": [
+                "aac",
+                "opus"
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "high",
+                "supported resolutions": [
+                    "3840×2160",
+                    "1920x1080",
+                    "1280x720",
+                    "852x480"
+                ],
+                "bitrate matrix": [
+                    {
+                        "res": "3840×2160",
+                        "fps": 30,
+                        "max bitrate": 14000
+                    },
+                    {
+                        "res": "1920x1080",
+                        "fps": 30,
+                        "max bitrate": 4500
+                    },
+                    {
+                        "res": "1280×720",
+                        "fps": 30,
+                        "max bitrate": 3072
+                    },
+                    {
+                        "res": "852x480",
+                        "fps": 30,
+                        "max bitrate": 1500
+                    }
+                ],
+                "max fps": 60,
+                "max video bitrate": 14000,
+                "max audio bitrate": 192,
+                "bframes": 2,
+                "x264opts": "tune=zerolatency scenecut=0"
```

Comment:
`tune=zerolatency` negatively impacts the quality more than it saves latency, especially if you're allowing b-frames you're better off not having it.